src/main/java/com/google/alpollo/model/SongInfo.java

```diff
@@ -0,0 +1,36 @@
+package com.google.alpollo.model;
+
+import java.util.List;
+import com.google.alpollo.Song;
+import com.google.alpollo.model.SongEntity;
+import com.googlecode.objectify.annotation.Index;
+import com.googlecode.objectify.annotation.Entity;
+
+/**
+ * Class used for storing data related to the Natural Language API analysis.
+ * It provides all the info related to a specific song.
+ */
+@Entity
+public class SongInfo {
```

Comment:
This class doesn't contain `id` field, without it this code won't compile, because you register this class in objectify. @sashapff knows how to fix it